internal/hud/renderer.go

```diff
@@ -217,22 +223,27 @@ func bestLogs(res view.Resource) string {
 		return string(res.LastBuild().Log)
 	}
 
-	// Two cases:
-	// 1) The last build finished before this pod started
-	// 2) This log is from an in-place container update.
-	// in either case, prepend them to pod logs.
-	if (res.LastBuild().StartTime.Equal(res.PodUpdateStartTime) ||
-		res.LastBuild().StartTime.Before(res.PodCreationTime)) &&
-		len(res.LastBuild().Log) > 0 {
-		return string(res.LastBuild().Log) + "\n" + res.PodLog
-	}
+	// TODO(matt) figure out what to do with DC logs once we have DC timestamps
```

Comment:
we've already got a type switch for DC further up in this function, put this todo up there

if it's NOT either DC or k8s, do we want to freak out?

Reply:
nope, it might be yaml